src/Resources/contao/widgets/ComponentStyleSelect.php

```diff
@@ -138,10 +138,24 @@ public function generate()
             $opts = StringUtil::deserialize($objStyleGroups->cssClasses);
 
             foreach ($opts as $opt) {
-                $arrFieldOptions[] = array(
+                $option = array(
                     'label' => $opt['value'] ?: $opt['key'],
                     'value' => $opt['key']
                 );
+
+                // skip third-party fields
```

Comment:
Change description to: `Dynamically change or expand group options`

Reply:
Done in b6e98db

Reply:
You have unfortunately adjusted the wrong line. 🙈